Part2-API-Schemas/openapi.yaml

```diff
@@ -206,6 +203,53 @@ components:
         - properties: 
             derivedFrom: 
               $ref: "https://api.swaggerhub.com/domains/Plattform_i40/Part1-MetaModel-Schemas/V3.0.1#/components/schemas/Reference"
+    AssetLink:
+      properties:
+        name:
+          type: string
+          minLength: 1
+          maxLength: 64
+          pattern: "^([\\t\\n\\r -\ud7ff\ue000-\ufffd]|\\ud800[\\udc00-\\udfff]|[\\ud801-\\udbfe][\\udc00-\\udfff]|\\udbff[\\udc00-\\udfff])*$"
+        value:
+          type: string
+          minLength: 1
+          maxLength: 2000
+          pattern: "^([\\t\\n\\r -\ud7ff\ue000-\ufffd]|\\ud800[\\udc00-\\udfff]|[\\ud801-\\udbfe][\\udc00-\\udfff]|\\udbff[\\udc00-\\udfff])*$"
+        externalSubjectId:
+          "$ref": "#/components/schemas/DeprecatedReference"
+        semanticId:
+          "$ref": "#/components/schemas/DeprecatedReference"
```

Comment:
why reference a "DeprecatedReference"?

Reply:
I really don't remember why we realized it this way. I need to double-check what works in Swaggerhub/OpenAPI and what not.